package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why do we use Git submodules instead of managing dependencies via npm?

Reply:
When you install this new module, we expect the following directory structure:

```
- node_modules
    - @cfn-modules
        - acm-certificate-public
            - node_modules
                - @cfn-modules
                    - lambda-function
```

Older versions of npm create this structure, but newer versions create this:

```
- node_modules
    - @cfn-modules
        - acm-certificate-public
        - lambda-function
```

Which works for nodejs but not for `aws cloudformation package`